pyproject.toml

```diff
@@ -23,10 +23,10 @@ readme = "README.rst"
 license = {file = "LICENSE"}
 requires-python = ">=3.8"
 dependencies = [
-    "scipy>=1.2.0",
-    "numpy",
+    "scipy",
+ "numpy",
```

Comment:
In the interests of historical completeness, could you put the (space-based) indentation back in?

Reply:
As someone who is a hard-tab enthusiast, I’m afraid this is a requirement and not just for historical completeness. The Python parsers are subject to the same limitations as the language, so mixing tabs and spaces is a hard error.

Reply:
Indentation is added back (with 3 spaces that seemed correct and doing a tab was too big of an indentation?)

Reply:
did you push this update?